recipes/mirotalk-sfu/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"id": "mirotalk-sfu",
+	"name": "MiroTalk SFU",
+	"version": "1.0.0",
+	"license": "MIT",
+	"config": {
+		"serviceURL": "https://sfu.mirotalk.com",
+    "hasCustomUrl": true,
+		"hasTeamId": false,
```

Comment:
please remove redundant/default configs

Reply:
Removed `hasTeamId` (as default false) added `"hasHostedOption": true,` as it can be self-hosted with a custom url.